### PR TITLE
Sungrid input from command file instead of command line

### PR DESCRIPTION
**Pull request by sha-ron**
Signed-off-by: Sharon Topaz <sharon.topaz@nuvoton.com>

Updating SGE to use a script that contain the execute command lines as input. 
Before SGE (qsub) got directly the execute command line with all the arguments on the same line.
There is a limit of 4096 characters in system buffer size. If the command line contains more than 4096 characters there is an error message:
  Error : Unable to run job: a path or filename may not exceed 4096 characters\nExiting.\n'
Updated the code so all our commands will be passed by script to qsub command.

### CODE REVIEW

File: util/dvsim/qsubopts.py [hunk 1]
```diff
@@ -1818,19 +1818,31 @@ def execute(self, mode='local', path=''):
...
-
-        exestring = ' '.join([program] + options + [self.args.command] + args)
+        # ---------------- command file -------------
+        cwd = os.getcwd()
+        command_file = cwd + '/command_file_' + str(os.getpid())
+        try:
+            with open(command_file, 'w') as f_command:
+                command_temp = str(self.args.command)
+                command_temp = command_temp.replace('"', '')
+                f_command.write(str(command_temp))
```
Comment (matutem) [inline util/dvsim/qsubopts.py:1828]: Why the extra "str" here?

**Review by matutem — APPROVED** [review]
Perhaps remove the extra "str" in line 1828, otherwise LGTM.

File: util/dvsim/qsubopts.py (hunk 1 above)
Comment (sha-ron) [reply util/dvsim/qsubopts.py]: Thanks for the review it is indeed redundant. Fixed.

**Comment by sha-ron** [comment]
Another small update so the log will be created correctly.

**Comment by sha-ron** [comment]
@matutem Can you please review ok and merge if it ok?

**Comment by sha-ron** [comment]
@matutem Reminder:)

**Comment by sha-ron** [comment]
Thanks @matutem. Can you please help me and merge? I don't permissions to merge.

**Comment by sha-ron** [comment]
@weicaiyang @msfschaffner Maybe you can help to merge please?

**Pull request merged**